automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/runtimes/RuntimeSystemSpec.scala

```diff
@@ -0,0 +1,104 @@
+import org.broadinstitute.dsde.workbench.leonardo.BillingProjectFixtureSpec.workspaceNameKey
+
+@DoNotDiscover
+class RuntimeSystemSpec extends RuntimeFixtureSpec {
+  implicit def ronToken: AuthToken = ronAuthToken.unsafeRunSync()(cats.effect.unsafe.IORuntime.global)
+
+  override val toolDockerImage: Option[String] = Some(LeonardoConfig.Leonardo.pythonImageUrl)
+
+  val dependencies = for {
+    storage <- google2StorageResource
+    httpClient <- LeonardoApiClient.client
+  } yield RuntimeGceSpecDependencies(httpClient, storage)
+
+  "RuntimeSystemSpec" - {
+    s"should have the workspace-related environment variables set in jupyter image" ignore { runtimeFixture =>
+      // TODO: any others?
+      val expectedEnvironment = Map(
+        "CLUSTER_NAME" -> runtimeFixture.runtime.clusterName.asString,
+        "RUNTIME_NAME" -> runtimeFixture.runtime.clusterName.asString,
+        "OWNER_EMAIL" -> runtimeFixture.runtime.creator.value,
+        "WORKSPACE_NAME" -> sys.props.getOrElse(workspaceNameKey, "workspace")
+      )
+
+      val res = dependencies.use { deps =>
+        implicit val httpClient = deps.httpClient
+        for {
+          runtime <- LeonardoApiClient.getRuntime(runtimeFixture.runtime.googleProject,
+                                                  runtimeFixture.runtime.clusterName
+          )
+          outputs <- expectedEnvironment.keys.toList.traverse(envVar =>
+            SSH.executeGoogleCommand(runtime.googleProject,
+                                     RuntimeFixtureSpec.runtimeFixtureZone.value,
+                                     runtime.runtimeName,
+                                     s"sudo docker exec -it jupyter-server printenv $envVar"
+            )
+          )
+        } yield outputs.map(_.trim).sorted shouldBe expectedEnvironment.values.toList.sorted
+      }
+      res.unsafeRunSync()(cats.effect.unsafe.IORuntime.global)
+    }
+
+    "should have Java available" ignore { runtimeFixture =>
+      val res = dependencies.use { deps =>
+        implicit val httpClient = deps.httpClient
+        for {
+          runtime <- LeonardoApiClient.getRuntime(runtimeFixture.runtime.googleProject,
+                                                  runtimeFixture.runtime.clusterName
+          )
+          output <- SSH.executeGoogleCommand(runtime.googleProject,
+                                             RuntimeFixtureSpec.runtimeFixtureZone.value,
+                                             runtime.runtimeName,
+                                             s"sudo docker exec -it jupyter-server java --version"
+          )
+        } yield output should include("OpenJDK Runtime Environment")
+      }
+      res.unsafeRunSync()(cats.effect.unsafe.IORuntime.global)
+    }
+
+    // TODO: remove selenium from this, needs re-write
+    // see https://broadworkbench.atlassian.net/browse/IA-4723
+    "should set up the users google identity on the vm" ignore { runtimeFixture =>
```

Comment:
This was something the hail spec was actually doing. Its an area that could deserve some discussion. Its a good test in my opinion, it essentially tests `when bucket permissions are as expected in the cloud (namely, when there is a file the vm user's pet has permission to access), I can use gsutil commands in the jupyter docker image to interact with it`. Unfortunately, it didn't work out of the box despite me essentially copying it from the hail spec, so I made a ticket to dig into it more and disabled it for now.

Reply:
Is this why the full class is marked as `@DoNotDiscover`?

Reply:
No, all specs in our suite are marked @DoNotDiscover, because we want to run them through the `LeonardoSuite`. Otherwise, sbt finds them when collecting the tests and they are run twice